README.md

```
@@ -1,61 +1,137 @@
 
-Ensure that you follow TDD approach to writing your code.
-The tests are available and marked as `skip`. Enable your tests by converting the
-`skip` into `test`
+1. Install dependencies:
+   ```bash
+   pnpm i
+   ```
 
-Ensure that your milestone tests are passing, before submitting the PR
+2. Run tests in watch mode:
+   To run the tests, first execute the command below. Then, in the test files, change `describe.skip` to `describe` and make the tests pass by writing the code (TDD).
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)